recipes/opencolorio/all/patches/1.1.1.patch

```diff
@@ -31,28 +31,92 @@ index e4f3119..1f7f77c 100644
  set(EXTDIST_BINPATH ${EXTDIST_ROOT}/bin)
  if(PYTHON_OK)
      set(EXTDIST_PYTHONPATH ${EXTDIST_ROOT}/${PYTHON_VARIANT_PATH})
-@@ -189,9 +191,9 @@ else(USE_EXTERNAL_TINYXML)
-         set(TINYXML_CMAKE_ARGS ${TINYXML_CMAKE_ARGS} -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TOOLCHAIN_FILE})
-     endif()
-     set(TINYXML_CMAKE_ARGS ${TINYXML_CMAKE_ARGS} -DCMAKE_BUILD_TYPE=${CMAKE_BUILD_TYPE})
+@@ -168,204 +170,20 @@ messageonce("Setting EXTDIST_PYTHONPATH: ${EXTDIST_PYTHONPATH}")
+ ### tinyxml ###
+ ##TODO: yaml and tinyxml : when there are not USE_EXTERNAL_TINYXML, use the same cmake schema/instructions => maybe refactorize in a cmake functions/macros ?
+ 
+-if(USE_EXTERNAL_TINYXML)
+-    set(TINYXML_VERSION_MIN "2.6.1")
+-    find_package(TinyXML)
+-    if(TINYXML_FOUND)
+-        if(TINYXML_VERSION VERSION_EQUAL ${TINYXML_VERSION_MIN} OR
+-           TINYXML_VERSION VERSION_GREATER ${TINYXML_VERSION_MIN})
+-            message(STATUS "External TinyXML will be used.")
+-        else()
+-            message(FATAL_ERROR "ERROR: ${TINYXML_VERSION} found, but ${TINYXML_VERSION_MIN} or newer is required.")
+-        endif()
+-    else(TINYXML_FOUND)
+-        message(STATUS "TinyXML was not found. Perhaps you forgot to install the development package?")
+-    endif(TINYXML_FOUND)
+-
```

Comment:
why not using `USE_EXTERNAL_TINYXML` branch instead of this big patch?

Reply:
Originally I followed the style of the original patch which removed similar code for yaml-cpp package, but I now use this block as suggested.